### ISSUE TITLE
Newest masterlist for Skyrim causes error message in LOOT 0.9.0

### ISSUE THREAD

**Issue by Lunamichelle47** [loot/loot#942]
Hello,

I am using LOOT 0.9.0 because it is the 'newest' version that can run on Vista.

After being unable to update my masterlist (error 12), I used the workaround from a previous problem resolution and downloaded it directly from https://github.com/loot/skyrim into my C:\User\AppData\Local\LOOT\Skyrim.  Unfortunately, upon starting LOOT, I see the following error:

An error occurred while parsing the masterlist: yaml-cpp: error at line 495, column 13: bad conversion: 'str' key missing from 'message content' object. This probably happened because an update to LOOT changed its metadata syntax support. Try updating your masterlist to resolve the error.

Additionally, the Masterlist Revision & Masterlist Date both say "Unknown: Git repository missing", which is probably because it is having a problem reading the masterlist.

P.S.: I saw a previous problem resolution saying that the newest masterlist could be found at https://raw.githubusercontent.com/loot/skyrim/v0.10/masterlist.yaml, however this presents the actual file contents and I am unable to download it as a file.  I did try copying and pasting the text to Notepad, but it freaked out because of the Unicode characters in the file.

System stats:

- OS: Vista SP2  (computer can't be updated to Windows 7-10 and I can't afford a new machine, so I'm stuck with Vista SP2 for the time being)

- LOOT: 0.9.0, plus I disabled the "update masterlist before sorting" setting because I keep getting error 12 when trying to update the masterlist.

- Newest Skyrim (not SE) and SKSE

I hope that this is enough information,  If you need more information, please let me know and I can provide it.

**Comment by Lunamichelle47** [comment]
Thank you!  :-)

**Comment by Lunamichelle47** [comment]
I just applied the 0.8 masterlist and the error no longer occurs, but the masterlist revision & date still say "unknown: Git repository missing".

Is this something I should worry about?  (I'm guessing not?)  Thanks again for a great tool (and great support too!)

**Comment by WrinklyNinja** [comment]
No, the repository is missing because you're using a file you downloaded instead of the masterlist updater.
